Travel-Planner/Travel-Planner/Screens/TripListView/TripListView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  TripListView.swift
+//  Travel-Planner
+//
+//  Created by siye yang on 10/2/24.
+//
+
+import SwiftUI
+
+struct TripListView: View {
+    
+    let cities = [
+            City(cityName: "Madrid", countryName: "Spain", imageName: "madrid"),
+            City(cityName: "Paris", countryName: "France", imageName: "paris"),
+            City(cityName: "Dubrovnik", countryName: "Croatia", imageName: "dubrovnik")
+        ]
+    
+    var body: some View {
+            ZStack(alignment: .topTrailing) {
+                // Main content
+                VStack(alignment: .leading) {
+                    // Title
+                    Text("Cities")
+                        .font(.largeTitle)
+                        .bold()
+                        .padding(.top, 40)
+                        .padding(.leading, 20)
+                    
+                    // ScrollView to hold the cards
+                    ScrollView {
+                        VStack(spacing: 20) {
```

Comment:
Use LazyVStack in scrollView

Reply:
done

Reply:
It's still VStack? Did you push your changes?